src/data/stat/unit_powerlaw.rs

```diff
@@ -11,9 +11,9 @@ use crate::traits::SuffStat;
 #[cfg_attr(feature = "serde1", serde(rename_all = "snake_case"))]
 pub struct UnitPowerLawSuffStat {
     /// Number of observations
-    n: usize,
+    pub n: usize,
```

Comment:
Make the members private or `pub(crate)` and give accessors iff we need them. I  believe that the `SuffStat` trait requires an `n` method anyway.